app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/controllers/TermsOfUseHistoryController.scala

```diff
@@ -117,16 +134,56 @@ class TermsOfUseHistoryController @Inject() (
       )
     }
 
-    def buildEmailSentModel(invite: TermsOfUseInvitation): TermsOfUseHistory = {
+    def buildEmailSentModelFromInvitation(invite: TermsOfUseInvitation): TermsOfUseHistory = {
       TermsOfUseHistory(
         DateTimeFormatter.ofPattern("dd MMMM yyyy").withZone(ZoneId.systemDefault()).format(invite.createdOn),
-        "Email sent",
-        "We invited admins of the application to agree to version 2 of the terms of use.",
+        deriveInvitationStatusDisplayName(EMAIL_SENT),
+        deriveInvitationStatusDescription(EMAIL_SENT),
+        None,
+        None
+      )
+    }
+
+    def buildReminderEmailSentModelFromInvitation(invite: TermsOfUseInvitation): TermsOfUseHistory = {
```

Comment:
Feels like some deduplication of code could happen here by passing in the date/time? I may be wrong.

Reply:
Yep, I agree - fixed.